nomad/grapl-core.nomad

```diff
@@ -0,0 +1,451 @@
+          ]
+        }
+      }
+
+      service {
+        name = "dgraph-alpha-${alpha.value.id}-grpc-private"
+        port = "${alpha.value.grpc_private_port}"
+        tags = ["dgraph", "alpha", "grpc"]
+
+        connect {
+          sidecar_service {
+            proxy {
+              # Connect to the Zero leader
+              upstreams {
+                destination_name = "dgraph-zero-0-grpc-private"
+                local_bind_port  = local.dgraph_zero_grpc_private_port_base
+              }
+
+              # Connect this Alpha to Zero followers
+              dynamic "upstreams" {
+                iterator = zero_follower
+                for_each = [for zero_follower in local.dgraph_zeros : zero_follower]
+
+                content {
+                  destination_name = "dgraph-zero-${zero_follower.value.id}-grpc-private"
+                  local_bind_port  = zero_follower.value.grpc_private_port
+                }
+              }
+
+              # Connect this Alpha to Other Alphas (but not to itself, obviously)
+              dynamic "upstreams" {
+                iterator = alpha_peer
+                for_each = [for alpha_peer in local.dgraph_alphas : alpha_peer if alpha_peer.id != alpha.value.id]
+
+                content {
+                  destination_name = "dgraph-alpha-${alpha_peer.value.id}-grpc-private"
+                  local_bind_port  = alpha_peer.value.grpc_private_port
+                }
+              }
+            }
+          }
+        }
+      }
+
+      service {
+        name = "dgraph-alpha-${alpha.value.id}-grpc-public"
+        port = "${alpha.value.grpc_public_port}"
+        tags = ["dgraph", "alpha", "grpc"]
+
+        connect {
+          sidecar_service {}
+        }
+      }
+
+      service {
+        name = "dgraph-alpha-${alpha.value.id}-http"
+        port = "${alpha.value.http_port}"
+        tags = ["dgraph", "alpha", "http"]
+
+        connect {
+          sidecar_service {}
+        }
+      }
+    }
+  }
+
+  group "grapl-graph-merger" {
+    count = var.num_graph_mergers
+
+    network {
+      mode = "bridge"
+    }
+
+    task "graph-merger" {
+      driver = "docker"
+
+      config {
+        image = "${var.container_registry}/grapl/graph-merger:${var.graph_merger_tag}"
+      }
+
+      env {
+        RUST_LOG           = "${var.rust_log}"
+        REDIS_ENDPOINT     = "${var.redis_endpoint}"
+        MG_ALPHAS          = "${local.alpha_grpc_connect_str}"
+        GRAPL_SCHEMA_TABLE = "${var.schema_table_name}"
+        AWS_REGION         = "${var.aws_region}"
+        # https://github.com/grapl-security/grapl/blob/18b229e824fae99fa2d600750dd3b17387611ef4/pulumi/grapl/__main__.py#L165
+        DEST_BUCKET_NAME      = "subgraphs-merged-bucket"
+        SOURCE_QUEUE_URL      = "${var.aws_sqs_url}/${var.aws_account_id}/graph-merger-queue"
+        DEAD_LETTER_QUEUE_URL = "${var.aws_sqs_url}/${var.aws_account_id}/graph-merger-dead-letter-queue"
```

Comment:
All the S3 bucket names, SQS queue URLs, and DynamoDB table names should all be passed in as variables, because of:

- https://github.com/grapl-security/grapl/pull/1072
- https://github.com/grapl-security/grapl/pull/1075
- https://github.com/grapl-security/grapl/pull/1076

The values present here right now wouldn't have worked anyway, because we used to prefix each one with the "deployment name". By just passing the full values, all the naming logic remains centralized in our infrastructure code, and all our services and deployment code (e.g., Docker Compose and Nomad files) can be a little bit simpler.

You won't need an `aws_sqs_url` or `aws_account_id` variable, though.

This applies to all other services in this file as well.

(I'd also recommend double-checking that these environments line up with what's in our Pulumi code for these services.)